src/test/regress/expected/multi_schema_support.out

```diff
@@ -546,16 +546,457 @@ SELECT master_create_worker_shards('test_schema_support.nation_hash_composite_ty
 -- insert some data to verify composite type queries
 \COPY test_schema_support.nation_hash_composite_types FROM STDIN with delimiter '|';
 SELECT * FROM test_schema_support.nation_hash_composite_types WHERE test_col = '(a,a)'::test_schema_support.new_composite_type;
- n_nationkey |          n_name           | n_regionkey |                      n_comment                      | test_col 
--------------+---------------------------+-------------+-----------------------------------------------------+----------
-           0 | ALGERIA                   |           0 |  haggle. carefully final deposits detect slyly agai | (a,a)
+ n_nationkey |          n_name           | n_regionkey |                     n_comment                      | test_col 
+-------------+---------------------------+-------------+----------------------------------------------------+----------
+           0 | ALGERIA                   |           0 | haggle. carefully final deposits detect slyly agai | (a,a)
 (1 row)
 
 --test with search_path is set
 SET search_path TO test_schema_support;
 SELECT * FROM nation_hash_composite_types WHERE test_col = '(a,a)'::new_composite_type;
- n_nationkey |          n_name           | n_regionkey |                      n_comment                      | test_col 
--------------+---------------------------+-------------+-----------------------------------------------------+----------
-           0 | ALGERIA                   |           0 |  haggle. carefully final deposits detect slyly agai | (a,a)
+ n_nationkey |          n_name           | n_regionkey |                     n_comment                      | test_col 
+-------------+---------------------------+-------------+----------------------------------------------------+----------
+           0 | ALGERIA                   |           0 | haggle. carefully final deposits detect slyly agai | (a,a)
 (1 row)
 
+-- test ALTER TABLE ADD/DROP queries with schemas
+SET search_path TO public;
+ALTER TABLE test_schema_support.nation_hash ADD COLUMN new_col INT;
+-- verify column is added
+\d test_schema_support.nation_hash;
+     Table "test_schema_support.nation_hash"
+   Column    |          Type          | Modifiers 
+-------------+------------------------+-----------
+ n_nationkey | integer                | not null
+ n_name      | character(25)          | not null
+ n_regionkey | integer                | not null
+ n_comment   | character varying(152) | 
+ new_col     | integer                | 
+
+\c - - - :worker_1_port
+\d test_schema_support.nation_hash_119*;
```

Comment:
Maybe, you can just check one shard instead of all
\d test_schema_support.nation_hash_1190003